README.md

```diff
@@ -63,3 +63,72 @@ And poof, you are ready to make a PR with the updates.
 ## Test environments
 
 Template projects are hooked up to Platform.sh projects, so each new PR gets built as a new, empty environment ready for testing.  In most cases simply visiting the built site and verifying that the installer can run (if available) or that the site gives the appropriate "there's nothing here yet" error is sufficient, but further testing can be done if needed.
+
+
+## Contributing
+
+If you would like to contribute to the list of Platform.sh's maintained templates, there is a simple process for ensuring that future maintenance is set up from the start through `template-builder`.
+
+Let's use the previous example: you have created a new application that uses the framework Spiffy that you think would be a useful template. 
+
+1. Create a repository under the [Platform.sh Github organization](https://github.com/platformsh/) for the template. Officially maintained templates begin with the `template-` prefix, so name it `template-spiffy`, and initialize it with an empty `README.md`.
+2. Add the "Examples and Templates team" as `Admin` collaborators in the `platformsh/template-spiffy` repository settings.
+3. Set up GitHub integration for the repository. When you submit your template for review it will helpful if there is already a live site that shows what the application would look like to customers and that it deploys properly. You can find information about how to set up a [GitHub integration](https://docs.platform.sh/administration/integrations/github.html#github) in the documentation.
+4. Clone the [`template-builder` repository](https://github.com/platformsh/template-builder) locally. Create and checkout a new branch called `add-spiffy`.
+5. Each template project is in its own directory within `templates`, which corresponds to a GitHub repository with the `template-` prefix. Create the directories `templates/spiffy` and `templates/spiffy/files/` on the `add-spiffy` branch. 
+6. Copy the files you want to be in `template-spiffy` into `templates/spiffy/files/`. If your application requires any patches to deploy on Platform.sh, copy them into `templates/spiffy/`.
```

Comment:
Since I give it 50/50 odds that the reader hasn't read the first half of the page, this point should emphasize that it's *just* the files that need to get *added*, and that the build process in Python should do as much of the work as possible, via downloading dependences, linking to an upstream, etc.  How much of that goes here vs deep-linking to code comments or the text above is debatable.